### ISSUE TITLE
Get started with Redis

### ISSUE THREAD

**Issue by danatinflux** [redis/redis-doc#2549]
Hey y'all, had a couple issues---could be user error, dunno. :)

I installed 'redis-stack' instead of 'vanilla redis' and the samples and configs aren't where the instructions say they should be. No harm, no foul---quick google search picked those up. 

However, what was breaking my brain is why the `/etc/init.d/redis_6379` wouldn't start. It gets created 644 and needs to have that +x bit set for it to work---so I did a `chmod +x etc/init.d/redis_6379` and it started like a champ. Thought some other folks might run into the same thing. 

Have a great day!

-dd

**Comment by chayim** [comment]
Thanks for letting us know! Can you help share what documentation you were looking at so that I can track this down and update accordingly? A link or two would be *great*!

**Comment by dmaier-redislabs** [comment]
@dwdougherty FYI Can you take a look at the documentation for the `samples and configs` please?

**Comment by danatinflux** [comment]
So sorry, I thought the workflow linked the page by default!

I was on the `https://redis.io/docs/getting-started/` page, and I was trying to follow these steps:

```
Copy the init script that you'll find in the Redis distribution under the utils directory into /etc/init.d. We suggest calling it with the name of the port where you are running this instance of Redis. For example:
```

I couldn't find a 'utils' dir or a file with that name, so I grabbed the file here:

```
https://github.com/pmem/pmem-redis/blob/master/utils/redis_init_script
```

Same thing with the redis conf file, ended up getting it from here:

```
https://redis.io/docs/management/config-file/
```

And then we look at the issue I had with this step:

![image](https://github.com/redis/redis-doc/assets/44617130/3c3515bd-6ad3-438b-96e3-a1c3b9ac9aba)

I had to `chmod` that file to get it to run under init.d...not sure if the 'proper' file that should have been present was supposed to be that way already?

These are the steps I used to install redis stack when I got started:

```
curl -fsSL https://packages.redis.io/gpg | sudo gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg
sudo chmod 644 /usr/share/keyrings/redis-archive-keyring.gpg
echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list
sudo apt-get update
sudo apt-get install redis-stack-server
```

Please let me know if there's anything else I can do to help! Have a great day!

**Comment by dwdougherty** [comment]
Thank you, Dan. That's very helpful. Which specific OS are you using?

**Comment by danatinflux** [comment]
Ubuntu 20:

```
techops@redis01:/$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.6 LTS
Release:	20.04
Codename:	focal
```

**Comment by dwdougherty** [comment]
The instructions in the **Install Redis more properly** section of the main **Get started** page read like they were originally written for folks installing Redis from source. I've added some notes to that section to call that out. After having gone through the slightly re-worked procedure, beginning with building from source, everything works smoothly. The init script (/path/to/redis-stable/utils/redis_init_script) included with the source distribution already has 0775 permissions, but I've also added a note to check that. Look for a PR soon.
